core/templates/pages/email-dashboard-pages/email-dashboard-data.service.ts

```diff
@@ -31,7 +31,7 @@ export class EmailDashboardDataService {
   // No. of query results to display on a single page.
   QUERIES_PER_PAGE: number = 10;
   // Store latest cursor value for fetching next query page.
-  latestCursor: string = null;
+  latestCursor: string | null = null;
```

Comment:
Add a comment why this can be `null`, ditto below.

Reply:
done. Below we are returning the same property